src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -107,6 +107,53 @@ private static Signers SignersFromJson(JArray _params)
             return ret;
         }
 
+        private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] args, Signers signers = null)
+        {
+            var snapshot = Blockchain.Singleton.GetSnapshot();
+            var contract = snapshot.Contracts.TryGet(scriptHash);
```

Comment:
If it's null, we can throw an error now

Reply:
I think signers should be allowed to be null, and some contract Verify methods don't necessarily require a signature.